signalr-node-server/signalr.js

```diff
@@ -39,12 +39,12 @@ class HubConnection {
         this._closeHandler = null;
 
         this._handshakeTimeout = setTimeout(() => {
-            if (!this._handshake) {
+            if (!this._handshake && this.connection) {
```

Comment:
failed sometimes when reconnecting / session ended, not sure but this fixed it.

Reply:
Hmm.